drivers/lis2dh12/lis2dh12_saul.c

```diff
@@ -31,8 +31,19 @@ static int read_accelerometer(const void *dev, phydat_t *res)
     return 3;
 }
 
+static int write_accelerometer(const void *dev, phydat_t *data)
+{
+    printf("Reg: 0x%02X\n",data->val[0]);
```

Comment:
Maybe I am missing some insight here: What is the purpose of these printf's? Is this function only for command line usage and this will be part of the output? If this is still for your debug purposes, please remove it.

Reply:
(I thought saul was also for non-console usage, but I never did actually use it.)